landscape/lib/tests/test_fs.py

```diff
@@ -64,12 +64,12 @@ def test_read_text_file(self):
 
     def test_read_text_file_with_limit(self):
         """
-        With a positive limit L{read_text_file} returns only the characters
-        after the first L{limit} characters as string.
+        With a positive limit L{read_text_file} returns up to L{limit}
+        characters from the start of the file.
         """
         utf8_content = codecs.encode(u"foo \N{SNOWMAN}", "utf-8")
         path = self.makeFile(utf8_content, mode="wb")
-        self.assertEqual(read_text_file(path, limit=3), u" ☃")
+        self.assertEqual(read_text_file(path, limit=3), u"foo")
```

Comment:
It'd be nice to have a test for a string with characters that encode to > 1 UTF-8 byte, and assert the correct behaviour of the decode with replace

Reply:
Resolved in ad801b2c93e560685d1722ed76375975688534d0